### ISSUE TITLE
Missing documentation on custom scenario

### ISSUE THREAD

**Issue by Galapagon** [maxsupermanhd/FactoCord-3.0#15]
The player messages are not including the up/down arrows or bold text for join/leave messages.
This could be because the env is not being read properly, though I believe these message controls were moved to control.lua

**Comment by maxsupermanhd** [comment]
Moved to scenario script. WIP on automatic map patching to enable this scenario for every save server will ever load.

**Comment by maxsupermanhd** [comment]
WIP on auto-append to scenario will be not added really soon, now only manual selection and patching. Decided after re-checked zip libraries available.

**Comment by Maximus-CZ** [comment]
4 years later I am experiencing same issue. Instead of `"**:arrow_up: {username}**"` I am getting plain `"{username} joined the game"` in discord.

How to fix?

**Comment by maxsupermanhd** [comment]
> I am getting plain "{username} joined the game" in discord.

Do you run a save with the control.lua we provide?

P.S. cool necro, please use discord instead

**Comment by Maximus-CZ** [comment]
no idea how to deal with control.lua? Is there a guide I missed somewhere? I cant find it in readme.

Not to be rude but I cant google answers in discord, but I could google this. Next person with the same problem might appreciate solution written down here, but if its gonna get answered in discord, its not gonna help him.

**Comment by maxsupermanhd** [comment]
Right, writing something more than 3 lines in INSTALL.md is probably something that should be done at some point.

In the save archive there is a scenario file called `control.lua` that you can customize, we provide example on what your scenario output could look like (https://github.com/maxsupermanhd/FactoCord-3.0/blob/master/control.lua). If you are familiar with modding of the save files and scenarios in general it should not be very difficult.

IIRC you can append it directly to existing scenario. (just don't append it twice or things will get very funny) What I was talking about 4 years ago was mechanism that would automatically patch a save file prior to server startup with custom formatting and all the bits and bobs for you but nobody made it and game in general went kinda quiet since everybody is waiting on expansion to be released. If you wish to make such functionality, pull requests are very welcome since I don't really have time nor will to dive into it right now no matter how easy it looks due to IRL reasons, Techno probably does not have time too.

**Comment by TechnoStrife** [comment]
As it turns out, `player_join` in the config file is not used for anything
